<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BotonCargarDicom">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>580</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar DICOM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonMostrarInfo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonCrearDicom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>580</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear DICOM</string>
    </property>
   </widget>
   <widget class="QLabel" name="Salida">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Esperando carga de archivos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Salida:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Plano Sagital</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plano Coronal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plano Axial</string>
    </property>
   </widget>
   <widget class="QSlider" name="SagitalSlider">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>339</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="SagitalSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="CoronalSlider">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="CoronalSpinBox">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="AxialSlider">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="AxialSpinBox">
    <property name="geometry">
     <rect>
      <x>832</x>
      <y>340</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="ListaInformacion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>671</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Info:</string>
    </property>
   </widget>
   <widget class="QWidget" name="GraficoSagital" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="GraficoCoronal" native="true">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>40</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="GraficoAxial" native="true">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>40</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="BarraProgreso">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="InfoAMostrar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
